.circleci/config.yml

```diff
@@ -198,24 +198,24 @@ references:
       when: on_fail
       command: |
         set +o errexit
-        if [[ $BRANCHNAME != "tests/"* ]]; then
+        if [[ $CIRCLE_BRANCH != "tests/"* ]]; then
           exit
         fi
         curl -X POST "https://a8c-gh-desktop-bridge.go-vip.co/circleciwebhook" \
         -H 'Cache-Control: no-cache'	\
         -H 'Content-Type: application/json'	\
         -d '{
-          "payload": {
-            "outcome": "'"failed"'",
-            "status": "'"failed"'",
-            "branch": "'"$BRANCHNAME"'",
-            "build_url": "'"$CIRCLE_BUILD_URL"'",
-            "build_parameters": {
-              "build_num": '"$CIRCLE_BUILD_NUM"',
-              "sha": "'"$sha"'",
-              "calypsoProject": "'"$calypsoProject"'"
+            "payload": {
+              "outcome": "'"failed"'",
+              "status": "'"failed"'",
+              "branch": "'"$BRANCHNAME"'",
+              "build_url": "'"$CIRCLE_BUILD_URL"'",
+              "build_parameters": {
+                "build_num": '"$CIRCLE_BUILD_NUM"',
+                "sha": "'"$sha"'",
```

Comment:
@bsessions85 Do these variables need to be exposed in the `mac` job the way they have been in the `build` job to-date?

https://github.com/Automattic/wp-desktop/blob/develop/.circleci/config.yml#L230-L233

If this isn't quite working yet, it's possible these env vars are not being populated because they haven't been exposed in the same way as prior in the `build` job.

Same would go for the `BRANCHNAME` var.

Reply:
yes, THANK YOU. I totally missed that.